src/templates/docs.tsx

```diff
@@ -60,12 +60,6 @@ const ContentContainer = styled.div<{ isZenMode: boolean }>`
   background-color: ${(props) => props.theme.colors.ednBackground};
 `
 
-const DesktopTableOfContents = styled(TableOfContents)<{
-  isPageIncomplete: boolean
-}>`
-  padding-top: ${(props) => (props.isPageIncomplete ? `5rem` : `3rem`)};
```

Comment:
To restate from the PR description, this ternary used here will no longer work, and instead the `isPageIncomplete` prop on the component is replaced with the Chakra prop `pt` and place the ternary there.